src/main/resources/google_checks.xml

```diff
@@ -61,7 +61,7 @@
         <module name="RightCurly">
             <property name="id" value="RightCurlyAlone"/>
             <property name="option" value="alone"/>
-            <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
+            <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT, LAMBDA"/>
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style
Looks like is clear by example on rightfully location.

@rnveach , please clarify your request.